x/genutil/legacy/v0_36/migrate.go

```diff
@@ -26,33 +24,6 @@ func Migrate(appState genutil.AppMap) genutil.AppMap {
 	v034gov.RegisterCodec(v034Codec)
 	v036gov.RegisterCodec(v036Codec)
 
-	// migrate genesis accounts state
-	if appState[v034genAccounts.ModuleName] != nil {
```

Comment:
Why was this removed?

Reply:
there's no genaccounts module anymore, it won't build